### PR TITLE
Change storing reports guide

### PR DESCRIPTION
**Pull request by IraTom**
The instructions for storing reports (as part of the multiple stands instruction) have been supplemented with a more detailed description of the actions.

### CODE REVIEW

File: docs/examples/multiple_stands.md [hunk 1]
```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
 
 ### storing reports
 
+We can use the database to store a list of reports. This will require a separate database.
+
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:68]: Here and below is need to remove extra empty lines.

File: docs/examples/multiple_stands.md [hunk 2]
```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
...
 We can use the **reports** database, as in the example [CouchDB Load](couchdb_load.md).
-A separate database is required to store the list of reports.
-By default, reports are saved to the default database on port 5984, but you can assign another port.
-Configure database on port 5986 for storing reports in `conftest.py` in the test folder:
 
-```python
-loader = CouchdbLoader(CouchdbConfig(port=5986))
+
+#### pytest.ini
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:72]: Duplicate information about the `.ini` file in the `###projects` chapter and here.

File: docs/examples/multiple_stands.md [hunk 3]
```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
...
+```
+
+#### conftest.py
+
+Create in the folder of each stand `conftest.py` file.
+
+Contains settings and fixtures for all tests:
+
+* The function of generating a report and recording it in the database save_report_to_couchdb;
+* The list of actions that will be performed after testing is filled in function fill_actions_after_test;
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:94]: Function names must be highlight by `` characters.

File: docs/examples/multiple_stands.md [hunk 4]
```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
...
+
+To store reports from all booths in one database, set the port number 
+of this database to a file `conftest.py ` in the folder of each stand.
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:97]: Extra space in `conftest.py`

File: docs/examples/multiple_stands.md [hunk 5]
```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
...
+from hardpy import (
+    CouchdbLoader,
+    CouchdbConfig,
+    get_current_report,
+)
+
+def save_report_to_couchdb():
+    report = get_current_report()
+    if report:
+        loader = CouchdbLoader(CouchdbConfig(port=5986))
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:111]: Why are we using 2 test projects with ports 5984 and 8985, but here we are using port 5986??

File: docs/examples/multiple_stands.md [hunk 6]
```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
...
+@pytest.fixture(scope="session", autouse=True)
+def fill_actions_after_test(post_run_functions: list):
+    post_run_functions.append(save_report_to_couchdb)
+    yield
 ```
 
 * `5986` - port of the database with reports
+
+
+#### Running CouchDB with Docker
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:123]: Duplicate information about the database in the `###databases` chapter and here.

File: docs/examples/multiple_stands.md (hunk 1 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: Extra empty lines have been removed.

File: docs/examples/multiple_stands.md (hunk 2 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: Deleted.

File: docs/examples/multiple_stands.md (hunk 4 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: Fixed.

File: docs/examples/multiple_stands.md (hunk 5 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md:78]: Port 5986 should be used for the third CouchDB Instance.

File: docs/examples/multiple_stands.md (hunk 6 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: Deleted.

File: docs/examples/multiple_stands.md [hunk 7]
```diff
@@ -19,13 +27,15 @@ import pytest
...
 ### databases
 
-Create a separate database for each project, and use a different port for each.
-To create a database use [CouchDH instance](../documentation/database.md#couchdb-instance) instruction.
+Create a separate CouchDB Instance for each project, and use a different port for each.
 
-* Create a `couchdb.ini` file:
+To create a CouchDB Instance use [CouchDH instance](../documentation/database.md#couchdb-instance) instruction.
+
+* Create a `couchdb.ini` file in each project.
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:38]: Where is the bash command for the third database on port 5986?

File: docs/examples/multiple_stands.md [hunk 8]
```diff
@@ -2,9 +2,17 @@
 
 This is an example of running multiple stands on one PC.
 
+The example implements the launch of two stands and three CouchDB Instances. 
+
+Each stand will record reports on the conducted testing in the CouchDB Instance assigned to it in the **runstore** and **statestore** databases.
+
+The third CouchDB Instance is needed to store reports in the **repot** database on each conducted test of each stand.
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:9]: **report**

File: docs/examples/multiple_stands.md [hunk 9]
```diff
@@ -43,34 +53,55 @@ For example, run a second database on port 5985:
...
+
 ### operator panels
 
-Configure in each project a connection to its own database for storing state and to its own operator panel.
+In projects with tests, configure a connection to its own database for storing state and to its own operator panel.
 To launch project with connection to database use script:
 
 ```bash
-hardpy-panel -dbp 5984 -wp 8000 tests
+hardpy-panel -dbp 5984 -wp 8000 .../tests
```
Comment (xorialexandrov) [inline docs/examples/multiple_stands.md:96]: `<dir_name>` instead of `...`

File: docs/examples/multiple_stands.md (hunk 8 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: fixed

File: docs/examples/multiple_stands.md (hunk 9 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: fixed

File: docs/examples/multiple_stands.md (hunk 4 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md]: fixed

File: docs/examples/multiple_stands.md (hunk 7 above)
Comment (IraTom) [reply docs/examples/multiple_stands.md:37]: Added.

**Review by xorialexandrov — APPROVED** [review]

**Pull request merged**